.coderabbit.yaml

```
@@ -0,0 +1,108 @@
+    - "!**/.ruff_cache/**"
+  path_instructions:
+    - path: "src/**/*.py"
+      instructions: |
+        Review Python source code focusing on:
+        - Code quality and maintainability
+        - Type annotations and mypy compatibility
+        - Security vulnerabilities and best practices
+        - Performance considerations
+        - Error handling and edge cases
+        - Async/await usage patterns
+    - path: "tests/**/*.py" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)